### ISSUE TITLE
Operating system font size not respected in tray menu

### ISSUE THREAD

**Issue by mannp** [nextcloud/desktop#2638]
I am a huge fan of Nextcloud and have been using it since the early versions, but I am disappointed the latest client (linux) appears to have completely disregarded accessibility?

The previous client had theme fonts, large easily readable, with the notification next to each nextcloud server icon, again easy to read with good contrast.

I have a top bar on the client window, so I can move it around.

Compared to the latest version 3.x I have tiny fonts that I don't appear to be able to change, so I have a full menu of things I never use as they are so tiny.

I am unable to move the client window, without using an alt key mouse move.

I always just go to settings to get the old client back.

Could the latest client have an accessibility re-check please?

Thanks for your consideration.

NC20.x Client Version 3.0.3 (Arch).

**Comment by er-vin** [comment]
It's a bit mentioning several issues in one. @jancborchardt opinions?

**Comment by mannp** [comment]
> It's a bit mentioning several issues in one?

Not sure what this means ....  can you elaborate?

**Comment by er-vin** [comment]
That you're raising more than one problem, this is OK it happens on some tickets but I prefer to put a note somewhere so that I remember.

Also asking opinions from our designer if he has ideas on how to move forward on some of the points.

**Comment by mannp** [comment]
> That you're raising more than one problem, this is OK it happens on some tickets but I prefer to put a note somewhere so that I remember.
> 
> Also asking opinions from our designer if he has ideas on how to move forward on some of the points.

Okay but the one problem is accessibility, I am no designer but the old interface uses my theme and the new one doesn't it dictates tiny fonts everywhere unless I go into settings and it presents me with the old interface.

If the new interface were to respect my theme it would be accessible.

A secondary point is there is no title bar on the client window to move the client around, but that would be back if the client respected my theme :+1:

**Comment by mannp** [comment]
> @mannp I changed the title to the main issue, which as I understand is that the system font size is not respected, correct?

@jancborchardt there are a number of related issues and simplifying to one of them doesn't for me capture the point.

1) The old client uses my theme fonts and sizes and is therefore readable to me and in line with the other applications on the desktop - The new client ignores all of that and presents tiny fonts only.

2) The old client has a top bar in line with my linux theme and allows me to move the client window around - the new one does not and cannot be moved.

3) Overall there is no consistency between the new client part and the old client part which is accessed when selecting the settings.

Only saying it is a font issue to me is only part of it...

Hope that helps?

**Comment by jancborchardt** [comment]
@mannp we need to keep separate issues separate though so we focus on _one_ problem per issue. :)

**Comment by mannp** [comment]
@jancborchardt then I'd stick with lack of accessibility :)

**Comment by FlexW** [comment]
I think we addressed some points of this issue:

You can have the main dialog as normal window and move it around if you either set `showMainDialogAsNormalWindow=true` in the section `[General]` in `nextcloud.cfg` or you disable the system tray.

The theming inconsistency will probably not change in the future because the main dialog is implemented with a different technology than the settings dialog.

I opened a separate ticket for the font issue https://github.com/nextcloud/desktop/issues/3797 to keep the issues focused. I will close that ticket for now.
